### PR TITLE
Use TEST_TIMEOUT-5 for timeout rather than hardcoded 60 sec.

### PR DESCRIPTION
**Pull request by inferno-chromium**
Fixes 2) in https://github.com/google/clusterfuzz/issues/1940.

### CODE REVIEW

File: src/python/bot/fuzzers/libfuzzer.py [hunk 1]
```diff
@@ -1537,11 +1537,13 @@ def remove_fuzzing_arguments(arguments, is_merge=False):
...
-  """Changes timeout argument for reproduction. This is higher than default to
-  avoid noise with smaller fuzzing defaults."""
+  """Changes timeout argument for reproduction. This is slightly less than the
+  |TEST_TIMEOUT| value for the job."""
   fuzzer_utils.extract_argument(arguments, constants.TIMEOUT_FLAG)
-  arguments.append(
-      '%s%d' % (constants.TIMEOUT_FLAG, constants.REPRODUCTION_TIMEOUT_LIMIT))
+
+  # Leave 5 sec buffer for report processing.
+  adjusted_test_timeout = max(1, environment.get_value('TEST_TIMEOUT') - 5)
```
Comment (Dor1s) [inline src/python/bot/fuzzers/libfuzzer.py:1545]: nit: would it make sense to move `5` to a constant too?

Comment (inferno-chromium) [reply src/python/bot/fuzzers/libfuzzer.py]: Done!

**Pull request merged**